code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/local/LocalAuthUtils.java

```diff
@@ -0,0 +1,33 @@
+package io.cattle.platform.iaas.api.auth.integration.local;
+
+import io.cattle.platform.core.constants.ProjectConstants;
+import io.cattle.platform.iaas.api.auth.TokenUtils;
+
+import java.util.List;
+
+public class LocalAuthUtils extends TokenUtils {
```

Comment:
Not that you have to change this, but most java developers expect classes that are named `*Utils` to just have static methods and not actually be instantiated.